### PR TITLE
[Feature]add multi matcher

### PR DESCRIPTION
**Pull request by justld**
torch.nn.MaxPool2d
torch.load
torch.save
torch.div
torch.split
torch.range
torch.meshgrid
torch.Tensor.is_contiguous
torch.Tensor.contiguous
torch.Tensor.copy_
torch.Tensor.masked_fill
torch.Tensor.unique
torch.Tensor.fill_
torch.Tensor.expand
torch.Tensor.requires_grad_
torch.Tensor.cuda
torch.utils.data.Dataset
torch.nn.functional.l1_loss
torch.nn.functional.binary_cross_entropy_with_logits
torch.nn.functional.max_pool2d
torch.distributed.init_process_group
torch.distributed.barrier
torch.distributed.max_memory_allocated

### CODE REVIEW

**Comment by justld** [comment]
paddle 结果：
max pool 2d:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-2.26860785]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.42342758]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.47053647])
max pool 2d:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-1.13085616]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.42342758]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.48092735])
max pool 2d:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-0.81852317]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.42342758]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.48450017])
max pool 2d:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-0.81852317]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.42342758]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.48450017])
torch div:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3433968.25000000]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [11125072.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.40641820])
torch div:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3433968.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [11125072.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.40655529])
torch div:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3433969.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [11125072.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.90636504])
torch div:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3433968.25000000]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [11125072.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.40641820])
torch div:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3433969.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [11125072.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.90636504])
torch split:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-5.08812475]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.33055592]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-0.00163324])
torch split:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-5.08812475]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.42342758]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-0.00042802])
torch range:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [10.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.50000000])
torch range:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [9.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.])
torch range:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [9.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.])
torch range:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [9.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.])
torch meshgrid:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [3.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [4.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [6.])
torch meshgrid:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [3.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [4.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [6.])
is_contiguous:  True
torch copy_:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-5.08812475]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.42342758]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-0.00037096])
masked_fill:  Tensor(shape=[1], dtype=int64, place=Place(gpu:0), stop_gradient=True,
       [5952775])
masked _fill:  Tensor(shape=[1], dtype=int64, place=Place(gpu:0), stop_gradient=True,
       [5952775])
torch unique:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3.92797184]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [3.83079410]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.00175083])
92099
torch unique:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3.92797184]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [3.83079410]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.00175083])
92099
torch unique:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3.92797184]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [3.83079410]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.00175083])
4
torch unique:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3.92797184]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [3.83079410]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.00171737]) [4, 10, 48, 48]
4
torch unique:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-3.92797184]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [3.83079410]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.00171737]) [4, 10, 48, 48]
torch fill_: Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [10.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [10.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [9.99999905])
expand:  [3, 3] Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.15569678]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.96558571]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.55737901])
torch cuda:  Place(gpu:0)
torch.nn.functional.l1_loss:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.12871349]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.12871349]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.12871349])
torch.nn.functional.l1_loss:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.12871349]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.12871349]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.12871349])
torch.nn.functional.l1_loss:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [8.15590286]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.12871349])
torch.nn.functional.l1_loss:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [13439817.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [13439817.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [13439817.])
torch.nn.functional.bce:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690])
torch.nn.functional.bce:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690])
torch.nn.functional.bce:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690])
torch.nn.functional.bce:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [9602405.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [9602405.]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [9602405.])
torch.nn.functional.bce:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.80643690])
max_pool2d:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-1.87069368]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.21305513]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.03004718]) [4, 50, 122, 122]
max_pool2d:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-1.87069368]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.21305513]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.03004718]) [4, 50, 122, 122]
max_pool2d:  Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [-1.80863512]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [5.09800577]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [1.03077877]) [4, 50, 81, 81]
I0317 03:32:29.855826 60724 tcp_utils.cc:181] The server starts to listen on IP_ANY:35152
I0317 03:32:29.856068 60724 tcp_utils.cc:130] Successfully connected to 172.17.0.5:35152
torch.cuda.max_memory_allocated:  429034496
torch softmax:   Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.00007508]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.70677465]) Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
       [0.02000000])
I0317 03:32:30.452600 60762 tcp_store.cc:257] receive shutdown event and so quit from MasterDaemon run loop
LAUNCH INFO 2023-03-17 03:32:32,113 Pod completed
LAUNCH INFO 2023-03-17 03:32:32,113 Exit code 0
